functions_lab.js

```diff
@@ -0,0 +1,313 @@
+ // ## Question 1
```

Comment:
Overall good demonstration of functions and scope. No major feedback or errors. Keep up the good work!